src/Symfony/Component/Console/Command/Command.php

```diff
@@ -203,7 +203,7 @@ protected function initialize(InputInterface $input, OutputInterface $output)
      *
      * @return int The command exit code
      *
-     * @throws \Exception
+     * @throws ExceptionInterface
```

Comment:
I think it would be event better to remove the annotation altogether as it does not bring anything really valuable.